.temp/test/main.ts

```diff
@@ -0,0 +1,3 @@
+const entry = import.meta.resolve("./worker")
```

Comment:
`The` main result of this PR should be that:

1. this argument rewritten to `"./worker.js"`, and
2. `worker.ts` is included in the call graph and transpiled even if that file is removed from `tsconfig.json`.

Once I get this working, I'll remove `.temp` and move it into a proper test.